packages/mobile/src/screens/personal-info/components/personal-info-form/personal-info-form.tsx

```diff
@@ -33,21 +29,22 @@ type Props = {
 
 const PersonalInfoForm: React.FC<Props> = ({ onSubmit }) => {
   const { t } = useTranslation();
+  const user = useAppSelector(selectCurrentUser);
   const { control, errors, handleSubmit } = useAppForm<UserPersonalInfo>({
     defaultValues: {
-      firstName: TEST_USER.firstName,
-      lastName: TEST_USER.lastName,
-      email: TEST_USER.email,
-      phone: TEST_USER.phone,
-      country: TEST_USER.country,
-      region: TEST_USER.region,
-      city: TEST_USER.city,
-      zipCode: TEST_USER.zipCode,
-      novaPoshta: TEST_USER.novaPoshta,
-      instagram: TEST_USER.instagram,
-      linkedin: TEST_USER.linkedin,
-      facebook: TEST_USER.facebook,
-      password: TEST_USER.password,
+      firstName: user?.firstName || '',
+      lastName: user?.lastName || '',
+      email: user?.email || '',
+      phone: user?.phone || '',
+      country: 'none',
+      region: 'none',
+      city: 'none',
```

Comment:
Can we set these to be empty strings by default too?

Reply:
yes, corrected